src/plugins/plugin-grpc.ts

```diff
@@ -469,26 +469,44 @@ function unpatchServer(server) {
 
 // # Exports
 
-var SUPPORTED_VERSIONS = '0.13 - 1';
-
 module.exports = [
   {
     file: 'src/node/src/client.js',
-    versions: SUPPORTED_VERSIONS,
+    versions: '0.13 - 1.6',
     patch: patchClient,
     unpatch: unpatchClient
   },
   {
     file: 'src/node/src/metadata.js',
-    versions: SUPPORTED_VERSIONS,
+    versions: '0.13 - 1.6',
     patch: patchMetadata,
     // patchMetadata doesn't modify the module exports of metadata.js.
     // So it's safe to have provide a no-op unpatch function.
     unpatch: function unpatchMetadata() {}
   },
   {
     file: 'src/node/src/server.js',
-    versions: SUPPORTED_VERSIONS,
+    versions: '0.13 - 1.6',
+    patch: patchServer,
+    unpatch: unpatchServer
+  },
+  {
+    file: 'src/client.js',
+    versions: '1.7',
+    patch: patchClient,
+    unpatch: unpatchClient
+  },
+  {
+    file: 'src/metadata.js',
+    versions: '1.7',
+    patch: patchMetadata,
+    // patchMetadata doesn't modify the module exports of metadata.js.
+    // So it's safe to have provide a no-op unpatch function.
+    unpatch: function unpatchMetadata() {}
```

Comment:
This duplicates the `unpatchMetadata()` function. Maybe define one at the top-level and use it from both places? Or use an arrow function instead? `unpatch: () => {}`

Reply:
Good catch -- removed duplicate comment as well